test/client/api.client.spec.unit.ts

```diff
@@ -35,6 +35,49 @@ describe("apiclient company profile unit tests", () => {
     }
     expect.assertions(1);
   });
+
+  it("correctly sets the overdue accounts flag if false in company profile but due date has passed",
+      async () => {
+    // Both here and in the tests below, create a deep copy of the dummy company profile before changing
+    // values, to ensure test isolation
+    const companyWithOverriddenAccountsNextDueDate: Resource<CompanyProfile> =
+        JSON.parse(JSON.stringify(dummySDKResponse));
+    companyWithOverriddenAccountsNextDueDate.resource.accounts.nextDue = "2019-05-31";
```

Comment:
Are you setting a fixed "current" date to test these dates against int he tests or just against the system time?

Reply:
No, but just seen that the extensions test does something like this to prevent tests which pass now from failing in the future, so will address this.

Reply:
Now setting the 'current' date, so this comment should now be addressed.